### PR TITLE
Dart sample [Pyramid Pattern]

### PR DESCRIPTION
**Pull request by ayushnishad**
add a link in Dart samples.

Please go throught [How to contribute guide](https://github.com/divyanshub024/awesome-dart-pad/blob/master/Contribution.md) before creating a PR.

### CODE REVIEW

**Comment by divyanshub024** [comment]
@ayushnishad Can you also add a screenshot of the output and come up with more accurate name like `pyramid pattern`.

**Comment by ayushnishad** [comment]
Okay.

**Comment by ayushnishad** [comment]
@divyanshub024 Done.

File: README.md [hunk 1]
```diff
@@ -76,3 +76,7 @@ If you appreciate the content 📖, support projects visibility, give 👍| ⭐|
   [<img src="/art/matchit.gif" width=240>](https://dartpad.dev/482dbd401031ab9f74b0fa3de705ad2f)
 
 ## Dart Samples
+
+- [Pattern](http://dartpad.dev/211761dbebee7c170cee8bc98d2314bc) by [Ayush Nishad](https://github.com/ayushnishad) (15/3/2020)
```
Comment (divyanshub024) [inline README.md]: Change the name here from `Pattern` to `Pyramid Pattern`.

Comment (ayushnishad) [reply README.md]: done.

**Comment by ayushnishad** [comment]
@divyanshub024  done.

**Review by divyanshub024 — APPROVED** [review]
LGTM!! Thanks a lot for this sample.

**Pull request merged**